k8s/upgrade/src/bin/upgrade-job/upgrade.rs

```diff
@@ -67,9 +60,8 @@ pub(crate) async fn upgrade(opts: &CliArgs) -> Result<()> {
         .await?;
 
     // Control plane containers are updated in this step.
-    if let Err(error) = helm_upgrade.run().await {
+    if let Err(error) = run_helm_upgrade.await {
```

Comment:
I don't know what we had a sleep, but why is it now needed now?

Reply:
The sleeps were there to let the event recorder finish it's job, I've removed it because shutdown_worker will handle those cases. We didn't have sleeps there, those were introduced recently.